### PR TITLE
Bump vk headers

### PR DESCRIPTION
**Pull request by RyzenDew**
Thanks to qurious For this fix!

The CI is failing because the vulkan headers version was bumped with commit https://github.com/RPCSX/rpcsx/commit/760aea0e49a6081405ba41151028a7166f90a223
it is building vulkan-headers 1.2.204 but requires minimum 1.3.247 for VK_EXT_SHADER_OBJECT
on my  build, adding the latest headers to rpcsx.yml succeeds.

### CODE REVIEW

File: .github/workflows/rpcsx.yml [hunk 1]
```diff
@@ -28,6 +28,15 @@ jobs:
         libglfw3-dev libvulkan-dev vulkan-validationlayers-dev \
         spirv-tools glslang-tools libspirv-cross-c-shared-dev \
         libxbyak-dev
+        
+        VULKANVER=1.3.259
+        curl -sL -o Vulkan-Headers.tar.gz https://github.com/KhronosGroup/Vulkan-Headers/archive/v${VULKANVER}.tar.gz
```
Comment (Margen67) [inline .github/workflows/rpcsx.yml:33]:
```suggestion
        curl -sSfLo Vulkan-Headers.tar.gz https://github.com/KhronosGroup/Vulkan-Headers/archive/v${VULKANVER}.tar.gz
```

File: .github/workflows/rpcsx.yml [hunk 2]
```diff
@@ -28,6 +28,15 @@ jobs:
         libglfw3-dev libvulkan-dev vulkan-validationlayers-dev \
         spirv-tools glslang-tools libspirv-cross-c-shared-dev \
         libxbyak-dev
+        
+        VULKANVER=1.3.259
```
Comment (Margen67) [inline .github/workflows/rpcsx.yml:32]:
```suggestion
        VULKANVER=1.3.259
```

File: .github/workflows/rpcsx.yml [hunk 3]
```diff
@@ -28,6 +28,15 @@ jobs:
...
+        curl -sL -o Vulkan-Headers.tar.gz https://github.com/KhronosGroup/Vulkan-Headers/archive/v${VULKANVER}.tar.gz
+        tar -xf Vulkan-Headers*.tar.gz
+        cd Vulkan-Headers*/ 
+        mkdir build && cd build
+        cmake ../ -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr
+        make
```
Comment (Margen67) [inline .github/workflows/rpcsx.yml:38]:
```suggestion
        cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr
        make -j$(nproc)
```

File: .github/workflows/rpcsx.yml [hunk 4]
```diff
@@ -28,6 +28,14 @@ jobs:
         libglfw3-dev libvulkan-dev vulkan-validationlayers-dev \
         spirv-tools glslang-tools libspirv-cross-c-shared-dev \
         libxbyak-dev
+        VULKANVER=1.3.259
+        curl -sSfLo Vulkan-Headers.tar.gz https://github.com/KhronosGroup/Vulkan-Headers/archive/v${VULKANVER}.tar.gz
+        tar -xf Vulkan-Headers*.tar.gz
+        cd Vulkan-Headers*/ 
+        mkdir build && cd build
+        cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr
```
Comment (Margen67) [inline .github/workflows/rpcsx.yml:36]:
```suggestion
        cd Vulkan-Headers*
        cmake . -Bbuild -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr
        cd build
```

**Comment by RyzenDew** [comment]
PR is done and ready to be merged.

**Pull request merged**